frontend/src/app/store/reducers/cases.ts

```diff
@@ -67,13 +94,20 @@ export const getAssessment =
 
       });
     };
-
-export const findAssessment =
-  (complaintIdentifier?: string): ThunkAction<Promise<string | undefined>, RootState, unknown, Action<string>> =>
-    async (dispatch) => {
+// Given a compaint id, returns the prevention education
+export const getPreventionEducation =
+  (complaintIdentifier?: string): AppThunk =>
```

Comment:
Should complaintIdentifier be optional? if it can be you might want to consider adding a guard to your get request to prevent it from failing. You also don't need the await on your get request if you're going to use the promise.